<launch>

    <node pkg="rnw_ros" type="rnw_odom_node" name="rnw_odom_node" output="screen"/>	  

    <node name="rosbag" pkg="rosbag" type="play" respawn="false" args="--start=12 --queue=1000 $(find rnw_ros)/bags/2020-05-27-00-28-56.bag" />

</launch>
